nwx_build_environment/add_catch2.dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM nwx_buildenv:latest
+
+ARG VERSION=2.13.8
```

Comment:
Can we get these version no. s from a single source?

Reply:
Eventually, yes. At the moment, not really. The image has to be built locally and pushed to the registry for the time being. We can worry about centralizing versions when we eventually automate building the image.

Reply:
> Eventually, yes. At the moment, not really. The image has to be built locally and pushed to the registry for the time being. We can worry about centralizing versions when we eventually automate building the image.

You mean currently instead of building the image during the process of CI, we have to build and push it manually before the CI process starts?

Reply:
The image is currently built manually, yes. But it doesn't have to be rebuilt before each CI run. It only has to be rebuild when changes or additions need to be made. Otherwise, the image should be fairly static.

Reply:
> The image is currently built manually, yes. But it doesn't have to be rebuilt before each CI run. It only has to be rebuild when changes or additions need to be made. Otherwise, the image should be fairly static.

Is it possible to create a workflow to update the image? This can be considered once our images are stable.

Reply:
Added a dispatchable workflow to build the image.